<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>749</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="documentMode" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <item row="0" column="0">
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="file_menu">
      <property name="title">
       <string>Файл</string>
      </property>
      <addaction name="add_files"/>
      <addaction name="separator"/>
      <addaction name="add_new_list"/>
      <addaction name="load_new_list"/>
      <addaction name="save_cur_list"/>
      <addaction name="separator"/>
      <addaction name="quit"/>
     </widget>
     <widget class="QMenu" name="window_menu">
      <property name="title">
       <string>Окно</string>
      </property>
      <addaction name="show_maximized"/>
     </widget>
     <widget class="QMenu" name="info_menu">
      <property name="title">
       <string>Справка</string>
      </property>
      <addaction name="action_4"/>
     </widget>
     <addaction name="file_menu"/>
     <addaction name="window_menu"/>
     <addaction name="info_menu"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,1,0,0,0,0">
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Показать только уникальные</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="utilsMenu" stretch="0,0,0,0,8,0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="playBtn_2">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../icons/play.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../icons/pause.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../icons/stop.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="trackSlider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="headphonesBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/open_headphones.svg</normaloff>
           <normalon>../icons/open_headphones.svg</normalon>../icons/open_headphones.svg</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="headphonesSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="footer">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="playBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../icons/play.svg</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="bootomSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QSplitter" name="SoundPadDiv">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="folders">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <column>
         <property name="text">
          <string>Наборы</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="sounds">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Номер</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Длительность</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Горячая клавиша</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Путь</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <action name="add_files">
   <property name="text">
    <string>Добавить звуковые файлы</string>
   </property>
  </action>
  <action name="add_new_list">
   <property name="text">
    <string>Новый список звуков</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Инструкция</string>
   </property>
  </action>
  <action name="load_new_list">
   <property name="text">
    <string>Загрузить список звуков</string>
   </property>
  </action>
  <action name="save_cur_list">
   <property name="text">
    <string>Сохранить свой список звуков</string>
   </property>
  </action>
  <action name="show_stats">
   <property name="text">
    <string>Показать общий счетчик воспроизведения</string>
   </property>
  </action>
  <action name="delete_stats">
   <property name="text">
    <string>Обнулить общий счетчик воспроизведения</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="show_maximized">
   <property name="text">
    <string>Максимизировать окно</string>
   </property>
  </action>
  <action name="show_minimizied">
   <property name="text">
    <string>Минимизировать окно</string>
   </property>
  </action>
  <action name="mix_tracks">
   <property name="text">
    <string>Перемешать список</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
